src/arkode/arkode_ls.c

```diff
@@ -2630,9 +2680,9 @@ int arkLsSolve(void* arkode_mem, N_Vector b, realtype tnow,
      set linear solver tolerance (in left/right scaled 2-norm) */
   if (arkls_mem->iterative) {
     deltar = arkls_mem->eplifac * eRNrm;
-    bnorm = N_VWrmsNorm(b, ark_mem->rwt);
+    bnorm = SUNCheckCallLastErrNoRet(N_VWrmsNorm(b, ark_mem->rwt));
     if (bnorm <= deltar) {
-      if (mnewt > 0) N_VConst(ZERO, b);
+      if (mnewt > 0) SUNCheckCallLastErrNoRet(N_VConst(ZERO, b));
```

Comment:
I realize that you said the automated tool would catch these, but I'm noting them manually just to be safe.
```suggestion
      if (mnewt > 0) {
        SUNCheckCallLastErrNoRet(N_VConst(ZERO, b));
      }
```